layer/layer_store.go

```diff
@@ -665,6 +665,10 @@ func (ls *layerStore) Cleanup() error {
 	return ls.driver.Cleanup()
 }
 
+func (ls *layerStore) DriverPut(id string) error {
```

Comment:
>layerstore should have a cleanup function that is run on daemon startup.
Thanks for your review.
how can we get a method to  exclude the correct mounts which are used by the running containers?